idl_templates.h

```diff
@@ -0,0 +1,5 @@
+//Web IDL doesn't seem to support C++ templates so this is the best we can do
+//https://stackoverflow.com/questions/42517010/is-there-a-way-to-create-webidl-bindings-for-c-templated-types#comment82966925_42517010
+typedef btAlignedObjectArray<btVector3> btVector3Array;
+typedef btAlignedObjectArray<btFace> btFaceArray;
+typedef btAlignedObjectArray<int> btIntArray;
```

Comment:
Thanks for the PR! The approach here looks good. One thought though, perhaps when we create new types like this we should avoid making them seem like native bullet ones, that is, not name them `btIntArray` but rather say `ammoIntArray`? That would clearly separate our additions from the core.

On the other hand, I can see it being more convenient for users to not have to think about that distinction, so maybe this PR is good as is.

Reply:
I don't mind the typedef names either way, as long as they have the needed functionality. In fact, in my [code](https://pastebin.com/3yqmdHLr) that actually uses these additions, I don't even mention btXXXArray by name. I just figured it made sense to name the new types after the originals. I can whip up a new PR if you want them changed.

Reply:
Yeah, good point, the type name really doesn't matter much in JS anyhow ;)